style/base.css

```diff
@@ -1,3 +1,7 @@
+h1 {
+  margin: 0;
+}
+
 p {
     font-family: 'IBM Plex Sans', sans-serif;
-}
\ No newline at end of file
+}
```

Comment:
these couple of whitespace changes ordinarily wouldn't be part of the PR, but we're missing newlines at the end of a couple of files, may as well fix it now